src/Analytics/DB/DataCollector.php

```diff
@@ -35,22 +47,23 @@ public function getSubscribedEvents()
     }
 
     /**
+     * Adds a participation to the data collector.
+     *
      * @param string $testIdentifier
      * @param string $variationIdentifier
      * @throws InvalidArgumentException
-     *
-     * @return void
```

Comment:
Not per se. It's just not needed. When no return type is provided, the default return type is void.